doc/source/tutorial/stats/probability_distributions.rst

```diff
@@ -0,0 +1,447 @@
+Probability distributions
+-------------------------
+
+.. toctree::
+    :maxdepth: 1
+
+    continuous
+    discrete
+    building_distributions
```

Comment:
Per another comment, `building_distributions` would be removed from this list and the content moved to the bottom of this document.

Can we include links to the `continuous` and `discrete` pages after there is some context like below?

Reply:
I have done both: moved the ToC for continuous and discrete distributions down for previous context, and also added the text you suggested. We can leave both or simplify, it's up to you. Thanks!